README.adoc

```diff
@@ -142,7 +142,7 @@ curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -
 
 == Editor Support
 
-`justfile` syntax is close enough to `make` that you may want to tell your editor to use make syntax highlighting for just.
+`justfile` syntax is close enough to `make` that you may want to tell your editor to use Make syntax highlighting for Just.
```

Comment:
Ditto here, I think `make` is clearer than Make.

Reply:
Done